package.json

```diff
@@ -15,10 +15,12 @@
     "clean": "rimraf dist storybook-static test",
     "publish-ghpages": "mkdir -p build-storybook && npm run build-storybook && gh-pages -d storybook-static",
     "cp": "cp package.json ./dist/package.json && cp README.md ./dist/README.md ",
-    "prepare": "npm run clean",
+    "prepare": "npm run clean && husky install",
     "publish-public": "npm run build && npm run cp && npm publish dist --access public",
     "publish-dry": "npm run build && npm run cp && npm publish dist --dry-run",
-    "release": "auto shipit"
+    "release": "auto shipit",
+    "pre-commit": "eslint --fix $(git diff --name-only --diff-filter=ACMRTUXB master | grep  -E \"(.js$)\") && git add .",
```

Comment:
right! I think this would work with a small change: `"DIFFED_FILES=$(git diff --name-only --diff-filter=ACMRTUXB --cached | grep -E \"(.js$)\"); eslint --fix $DIFFED_FILES && git add $DIFFED_FILES"`, otherwise it will fail if the files you've changed on the commit are all non .js

Reply:
Good spot! Command line isn't my strong point :)